eclair-core/src/main/scala/fr/acinq/eclair/payment/receive/MultiPartHandler.scala

```diff
@@ -223,14 +229,25 @@ object MultiPartHandler {
               val paymentPreimage = paymentPreimage_opt.getOrElse(randomBytes32())
               val paymentHash = Crypto.sha256(paymentPreimage)
               val expirySeconds = expirySeconds_opt.getOrElse(nodeParams.paymentRequestExpiry.toSeconds)
-              val features = {
-                val f1 = Seq(Features.PaymentSecret.mandatory, Features.VariableLengthOnion.mandatory)
-                val allowMultiPart = nodeParams.features.hasFeature(Features.BasicMultiPartPayment)
-                val f2 = if (allowMultiPart) Seq(Features.BasicMultiPartPayment.optional) else Nil
-                val f3 = if (nodeParams.enableTrampolinePayment) Seq(Features.TrampolinePayment.optional) else Nil
-                PaymentRequest.PaymentRequestFeatures(f1 ++ f2 ++ f3: _*)
+              val paymentMetadata = Some(ByteVector64.Zeroes.bytes)
```

Comment:
I would use the smallest possible vector, since our BOLT 11 invoices are already too big:
```suggestion
              val paymentMetadata = Some(ByteVector(0))
```

Reply:
It's a trade-off: when we'll want to actually use this, we will probably need 32 to 64 bytes to do DB-less invoices, so shouldn't we test that sender are able to send that much?

Reply:
Ah! 🤔 Well I think that here we are just testing that the sender _supports_ the metadata, nothing more. Going further and trying to validate additional requirements (that we are not even sure of?) is a different concern imo. Debatable...

Reply:
That's true, maybe we should wait to have a scheme that we want to use with `payment_metadata` (most likely to get rid of invoice storage until they're paid), see how many bytes that uses, and at that point set a payment metadata of that length?

Then after X months if senders are able to send that many bytes, we can actually use the designed scheme.

Reply:
Updated in https://github.com/ACINQ/eclair/pull/2063/commits/c2daf43855f1effb147119f8451b9f239f2e7548, we now set a single byte metadata.